src/Engine/Graphics/Renderer/OpenGLRenderer.cpp

```diff
@@ -5052,10 +5046,9 @@ void OpenGLRenderer::DrawTwodVerts() {
     twodshader.use();
 
     // set sampler to palette
-    glUniform1i(twodshader.uniformLocation("palbuf"), GLint(1));
-    glActiveTexture(GL_TEXTURE0 + 1);
-    glBindTexture(GL_TEXTURE_BUFFER, paltex);
-    glTexBuffer(GL_TEXTURE_BUFFER, GL_RGBA8UI, palbuf);
+    glBindTexture(GL_TEXTURE_2D, paltex2D);
```

Comment:
glActiveTexture(GL_TEXTURE0 + paltex2D_id);
glBindTexture(GL_TEXTURE_2D, paltex2D);

and here to make sure we bind to the right unit